packages/arb-validator-core/utils/wallet.go

```diff
@@ -0,0 +1,88 @@
+package utils
+
+import (
+	"flag"
+	"fmt"
+	"math"
+	"math/big"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+type WalletFlags struct {
+	passphrase *string
+	gasPrice   *float64
+}
+
+func AddFlags(fs *flag.FlagSet) WalletFlags {
+	passphrase := fs.String("password", "", "password=pass")
+	gasPrice := fs.Float64("gasprice", 4.5, "gasprice=FloatInGwei")
+
+	return WalletFlags{
+		passphrase: passphrase,
+		gasPrice:   gasPrice,
+	}
+}
+
+func GetKeystore(validatorFolder string, args WalletFlags, flags *flag.FlagSet) (*bind.TransactOpts, error) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 109 characters, out of limit 80](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)